handlers/name_handler.py

```
@@ -0,0 +1,81 @@
+        except HTTPException as e:
+            raise
+        except Exception as e:
+            raise HTTPException(
+                status_code=500, detail=ERROR_RESPONSES["SERVER_ERROR"]
+            ) from e
+    
+    def change_name(self, used_id: str, new_name: str):
+        try:
+            updated_name = NameUseCase.change_name(used_id, new_name)
+            return {"message": "이름이 수정되었습니다", "updated_name": updated_name}
+        except HTTPException as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)